src/examples/source/GUI.cpp

```diff
@@ -381,8 +381,9 @@ int main(int argc, char** argv)
     }
     if (ImGui::BeginPopupModal("About modal", NULL, ImGuiWindowFlags_AlwaysAutoResize))
     {
+      // TODO: read text from file and print in the window
```

Comment:
Would be better to add a task in JIRA than getting a TODO inside the code, for this kind of remark. TODO goes well for missing states handling, or identified performance. everything that would help someone debugging there. features should go to to JIRA. This is minor :)

Reply:
This TODO is more of a note for a future when we decide to make GUI modular - at the moment it can stay as it is